README.md

```diff
@@ -77,7 +77,12 @@ For sync communication we have an IRC channel #CRI-O, on chat.freenode.net, that
 
 ## Getting started
 
-### Prerequisites
+### Runtime dependencies
+
+- runc
```

Comment:
Make it `runc or cc-runtime or any OCI compatible runtime`

Reply:
awesome yeah

Reply:
done!